contracts/governance/src/governance.rs

```diff
@@ -0,0 +1,296 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) DUSK NETWORK. All rights reserved.
+
+use core::iter;
+
+use alloc::vec;
+use alloc::vec::Vec;
+use canonical_derive::Canon;
+use dusk_bls12_381::BlsScalar;
+use dusk_bls12_381_sign::{PublicKey as BlsPublicKey, Signature};
+use dusk_bytes::Serializable;
+use dusk_pki::PublicKey;
+
+#[cfg(not(feature = "map"))]
+use crate::collection::Collection;
+
+use crate::*;
+
+#[cfg(feature = "map")]
+type Collection<K, V> = dusk_hamt::Map<K, V>;
+
+#[derive(Debug, Default, Clone, Canon)]
+pub struct GovernanceContract {
+    pub(crate) seeds: Collection<BlsScalar, ()>,
+    pub(crate) balances: Collection<PublicKey, u64>,
+    pub(crate) whitelist: Collection<PublicKey, ()>,
+    pub(crate) paused: bool,
+    pub(crate) total_supply: u64,
+}
+
+impl GovernanceContract {
+    /// Authority of the contract
+    ///
+    /// Will have to be defined in the constant space so the bytecode of the
+    /// contract will be changed as the authority does
+    pub const AUTHORITY: &[u8; 96] =
+        include_bytes!(concat!(env!("RUSK_PROFILE_PATH"), "/2EM4m2NvFMi9sn7FGVDPGube2APvv9BzHV6HH79LGFouAXHdBQPq8ezpiKfBdPpZotwwJPXK3vS8mu6fH3Tn94un.cpk"));
```

Comment:
Shouldn't this have a friendlier name? like `env!("RUSK_PROFILE_PATH"), "/governance_authority.cpk"`